applications/nrf5340_audio/unicast_client/Kconfig.defaults

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config BT_BAP_UNICAST_CLIENT
+	default y
+config BT_ISO_TX_BUF_COUNT
+	default 2
+config BT_MAX_CONN
+	default 2
+
+# Generic Audio Source - 0x0880
+config BT_DEVICE_APPEARANCE
+	default 2176
+config BT_ISO_MAX_CHAN
+	default 4
+config BT_MAX_PAIRED
+	default 2
+config BT_BAP_UNICAST_CLIENT_GROUP_STREAM_COUNT
+	default 4
+config BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT
+	default 2
+config BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT
+	default 2
+config BT_VCP_VOL_CTLR
+	default y
+config BT_MCS
+	default y
+config BT_GATT_DYNAMIC_DB
+	default y
+config UTF8
+	default y
+config BT_MPL
+	default y
+config MCTL
+	default y
+config MCTL_LOCAL_PLAYER_CONTROL
+	default y
+config MCTL_LOCAL_PLAYER_REMOTE_CONTROL
+	default y
+config BT_AUDIO_TX
+	default y
+config BT_AUDIO_RX
+	default y
```

Comment:
Should we check if STREAM_BIDIRECTIONAL in here?

Reply:
No, my intention was that this is a generic unicast_client and all features will be enabled and then it is up to the unicast_server if it's going to be bidirectional or not